src/bin/pgcopydb/pgcmd.c

```diff
@@ -350,8 +350,9 @@ pg_dump_db(PostgresPaths *pgPaths,
 
 	SafeURI safeURI = { 0 };
 	bool pgpassword_found_in_env = env_exists("PGPASSWORD");
-	char PGPASSWORD[MAXCONNINFO] = { 0 };
-
+	size_t len = strlen(pguri) + 1;
+	char *PGPASSWORD = (char *) calloc(len, sizeof(char));
```

Comment:
The call to `pg_env_dup` is going to take care of memory allocation by calling into `strdup` for us. So that piece of memory is wasted. Just set `char *PGPASSWORD = NULL;`.